docs/getting-started/triggers.yaml

```diff
@@ -0,0 +1,101 @@
+apiVersion: tekton.dev/v1alpha1
+kind: TriggerTemplate
+metadata:
+  name: getting-started-triggertemplate
+  namespace: getting-started
+spec:
+  params:
+    - name: gitrevision
+      description: The git revision
+      default: master
+    - name: gitrepositoryurl
+      description: The git repository url
+    - name: namespace
+      description: The namespace to create the resources
+  resourcetemplates:
+    - apiVersion: tekton.dev/v1alpha1
+      kind: PipelineResource
+      metadata:
+        name: source-repo-$(uid)
+        namespace: $(params.namespace)
+        labels:
+          triggertemplated: "true"
+      spec:
+        type: git
+        params:
+        - name: revision
+          value: $(params.gitrevision)
+        - name: url
+          value: $(params.gitrepositoryurl)
+    - apiVersion: tekton.dev/v1alpha1
+      kind: PipelineResource
+      metadata:
+        name: image-source-$(uid)
+        namespace: $(params.namespace)
+        labels:
+          triggertemplated: "true"
+      spec:
+        type: image
+        params:
+          - name: url
+            value: DOCKERREPO-REPLACEME # docker-repo-location.com/repo:getting-started
+    - apiVersion: tekton.dev/v1alpha1
+      kind: PipelineResource
+      metadata:
+        name: event-to-sink-$(uid)
+        namespace: $(params.namespace)
+        labels:
+          triggertemplated: "true"
+      spec:
+        type: cloudEvent
+        params:
+          - name: targetURI
+            value: http://event-display.getting-started.svc.cluster.local
+    - apiVersion: tekton.dev/v1alpha1
+      kind: PipelineRun
+      metadata:
+        name: getting-started-pipeline-run-$(uid)
+        namespace: $(params.namespace)
+        labels:
+          triggertemplated: "true"
```

Comment:
All resources now have the labels [specified here](https://github.com/tektoncd/triggers/blob/master/docs/triggertemplates.md). As a result we can remove this label.

Reply:
sgtm 👍